source/blog/2016-08-20-building-compassionate-software.html.markdown

```diff
@@ -0,0 +1,143 @@
+2. **Teams with Psychological Safety Perform Better.** I'm going to describe what psychological safety means and what it looks like.
+3. **How to Implement Psychological Safety on Your Team.** After we have a firm grasp on feelings and psychological safety, I want to discuss some ways to start improving your team's performance and dynamics.
+
+I've had this topic on my mind for a long time, and the 
+
+## Feelings Matter
+
+So I've [written about this before](https://ashfurrow.com/blog/empathetic-civilization/), but I'll say it again: _feelings matter_. It might sound obvious to you, but it really does need to be repeated. Compassionate software can't be built without compassion for each other.
+
+Feelings matter, a lot. [We've actually researched this](http://qz.com/622749/teens-do-better-in-science-when-they-know-einstein-and-curie-also-struggled/): students who learned about the struggles that scientists went through on their way to achieving success did a lot better in science class. And students who _didn't_ learn did started doing _worse_.
+
+Learning how successful scientists struggle helped students when they inevitably struggled. That's because struggling is normal, but when we neglect to mention the struggles of history's great scientists, we present the incorrect view that they just _were great_. And that's not true, [everyone struggles](https://ashfurrow.com/blog/normalizing-struggle/) sometimes.
+
+When students see themselves and their own struggles represented in the history of science, they learn to _empathize_ with scientists. Empathy, the core of emotions, is the practice of sharing another person's point of view and feelings.
+
+BEGIN_WIDE
+
+YOUTUBE 1Evwgu369Jw
+
+END_WIDE
+
+Empathy is a choice we make. In 1996, [Theresa Wiseman's work](http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2648.1996.12213.x/full) ([PDF](https://www.researchgate.net/profile/Theresa_Wiseman/publication/227941757_A_concept_analysis_of_empathy/links/00b4951e409deac34f000000.pdf)) explained that there are four components to empathy:
+
+- Seeing the world as others see it.
+- Recognizing and understanding another's feelings.
+- Staying non-judgemental.
+- Communicating that you understand.
+
+These are things we can do, things we can _choose_ to do. We can practice, we can set a Google Calendar reminder to reflect on recent meetings with the intent to empathize with our peers.
+
+This is how someone empathizes, which is core to feelings, which matter. Feelings are important. Now I want to explore what it means to be a member of a team where feelings are prioritized.
+
+## Teams with Psychological Safety Perform Better
+
+So you want to be a 10x developer, eh? You may have heard that the 10x developer is a myth, but that's not true: a 10x developer is someone who makes the five developers around them each twice as productive. You can be a 10x developer by making sure that your team has psychological safety.
```

Comment:
Not sure who said it first, but I saw it in a tweet.

https://twitter.com/peterseibel/status/512615519934230528